src/main/scala/models/ApplicationForm.scala

```diff
@@ -25,3 +26,35 @@ case class ApplicationFormSection(sectionNumber: Int, title: String, questions:
 }
 
 case class ApplicationForm(id: ApplicationFormId, opportunityId: OpportunityId, sections: Seq[ApplicationFormSection])
+
+trait ApplicationFormSectionType {
```

Comment:
This should be sealed trait to enable compiler checking pattern matching completeness